--- pacman-6.0.2/lib/libalpm/util.c	2022-10-03 04:52:04.000000000 +0200
+++ util.c	2023-09-10 17:12:06.592162670 +0200
@@ -583,7 +583,7 @@
  * @param stdin_ctx context to be passed to @a stdin_cb
  * @return 0 on success, 1 on error
  */
-int _alpm_run_chroot(alpm_handle_t *handle, const char *cmd, char *const argv[],
+int SYMEXPORT _alpm_run_chroot(alpm_handle_t *handle, const char *cmd, char *const argv[],
 		_alpm_cb_io stdin_cb, void *stdin_ctx)
 {
 	pid_t pid;
